### PR TITLE
Integrated board hardware

### PR DESCRIPTION
**Pull request by doudar**
This change introduces the newest PCB with a serial interface specifically designed for integration with Peloton bikes and will provide a path for DIY'ers to order a board and cable kit that doesn't require any soldering skills. It's "print", "plug" and "play"

### CODE REVIEW

**Review by eMadman — CHANGES_REQUESTED** [review]
TODO: 

- [x] hyperlinks in readme.md don't work - relative links aren't working in our favor.
- [x] Readme.md for the main /hardware folder - needs a folder directory and descriptions to direct users.
- [ ] Rename files in the arm folder to their length measured across a common section.
- [ ] Add a table to readme.md in this folder to cross reference arm length to bikes.

File: Hardware/README.md [hunk 1]
```diff
@@ -1,7 +1,40 @@
-Version 2 is new, prints easier, is smaller and uses less hardware. It is currently only compatable with the PCB.  
+# [SmartSpin2K Hardware]
```
Comment (doudar) [inline Hardware/README.md:1]: This file looks great!

**Comment by doudar** [comment]
If .\Hardware\Common Assets\Arm\CAD\Bowflex C7.f3d from integrated board hardware is different from the current develop, we will need to manually add it back in after preforming this merge. I saved a local copy.

**Comment by eMadman** [comment]
> If .\Hardware\Common Assets\Arm\CAD\Bowflex C7.f3d from integrated board hardware is different from the current develop, we will need to manually add it back in after preforming this merge. I saved a local copy.

Version on Develop is the most recent

**Comment by eMadman** [comment]
My todos are not yet done but can move into a new PR.

**Review by eMadman — APPROVED** [review]

**Pull request merged**